packages/graph/hash_graph/lib/graph/src/api/mod.rs

```diff
@@ -1 +1,2 @@
 pub mod rest;
+pub mod utoipa;
```

Comment:
I think that would make sense as we're using `utoipa` directly in `rest`, yeah

Reply:
This is not as easy as said. I had id in the `rest` module first but the derive macros of utoipa are not smart enough to refer to the external crate `::utoipa` but refer to the local module then instead. I see three options:
- Rename the `utoipa` module to something different (`helper`? `serialized`?)
- Move all usages of `::utoipa` out of `crate::api::rest`
- Leave it at that location
I chose the last option as it was the easiest and we eventually will remove utoipa. Happyy to apply other changes.

Reply:
>  and probably renamed to something like typedefs.

I also asked for us to rename it from utoipa. We have agreed before we don't like using external crate names as module names. I was going to block the PR on it being renamed, so that should solve your problem :)

`api::rest::typedef` seems fine to me

Reply:
Moved to `api::rest::utoipa_typedef` 3ba34c780b8c39d05b9a08e78f767f5ac8dbb367